lib/rewrite.js

```diff
@@ -293,15 +288,12 @@ exports.search = function(js, searchRule) {
   // remove if one exists as the first line
   js = helpers.removeShebang(js);
 
-  var pattern = unwrapNode(esprima.parse(searchRule, {
+  var pattern = unwrapNode(parser.parse(searchRule, {
     raw: true
   }));
 
   var matches = [];
-  falafel(js, {
-    raw: true,
-    loc: true
```

Comment:
https://github.com/rdio/jsfmt/blob/e35a6ae310e653fdfeb86c2bfdee724729d793df/lib/run.js#L116

It is used by `jsfmt --search 'a' ./file.js`